Sources/SwiftParser/TypeAttribute.swift.gyb

```diff
@@ -0,0 +1,29 @@
+%{
+  from gyb_syntax_support import *
+  from gyb_syntax_support.AttributeKinds import *
+  # -*- mode: Swift -*-
+  # Ignore the following admonition it applies to the resulting .swift file only
+}%
+//// Automatically Generated From DeclarationAttribute.swift.gyb.
+//// Do Not Edit Directly!
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+@_spi(RawSyntax) import SwiftSyntax
+
+extension Parser {
```

Comment:
Why did you choose to define `TypeAttribute` as a nested type in `Parser`? I think they could be equally valuable e.g. for `Lookeahead` and I would define them as top level types.

(same for `DeclarationAttribute`)

Reply:
I'll move it in a follow-up. I want to scope this just to the python.

Reply:
What do you mean by “I want to scope this just to the python.”

Reply:
These types are nested in extensions of the Parser type before this PR, I left them nested afterwards.

Reply:
Ah, I see. Makes sense 👍